### DISCUSSION TITLE
Gruppen Aktion Rolladen

### DISCUSSION THREAD

**Discussion by holydevil72** [Zefau/ioBroker.jarvis#2343] [Q&A]
Ist da eine Möglichkeit die Gruppenaktion welche es ja auch für die Beleuchtung gibt auf Rolläden ebenfalls anzuwenden ?

**Comment by mcuiobroker** [comment]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/geraet/actionelement/blindlevelaction#mehrere-rolllaeden-blinds-gleichzeitig-steuern
